services/motion/builtin/builtin.go

```diff
@@ -274,22 +274,47 @@ func (ms *builtIn) MoveOnGlobe(
 	if err != nil {
 		return false, err
 	}
+	currentPosition := currentPIF.Pose().Point()
+
+	// get position of localizer relative to base
+	robotFS, err := ms.fsService.FrameSystem(ctx, nil)
+	if err != nil {
+		return false, err
+	}
+	localizerFrame := robotFS.Frame(movementSensorName.ShortName())
+	if localizerFrame != nil {
+		localizerParent, err := robotFS.Parent(localizerFrame)
```

Comment:
This is not the way to do this. Not only would this currently give the localizer's origin frame rather than the base frame, but even if those weren't a thing, there's no guarantee that the base is the localizer's parent. Or that there are 0dof between the localizer and the base.

This should request current inputs for the frame system, and use `robotFS` to request the transform betwene the localizer and the base at call time.